tox.ini

```diff
@@ -1,29 +1,53 @@
 [tox]
 min_version = 4.0
 envlist =
-    py{310,311,312}{,-oldestdeps,-devdeps,-online,-figure,-conda}
-    build_docs
+    py{310,311,312}
+    py312-devdeps
+    py310-oldestdeps
     codestyle
+    build_docs
 requires =
-    tox-pypi-filter
+    setuptools
+    pip
+    tox-pypi-filter>=0.14
 
 [testenv]
 pypi_filter = https://raw.githubusercontent.com/sunpy/sunpy/main/.test_package_pins.txt
-allowlist_externals=
+# Run the tests in a temporary directory to make sure that we don't import
+# the package from the source tree
+allowlist_externals =
     /bin/bash
     /usr/bin/bash
-changedir = .tmp/{envname}
+change_dir = .tmp/{envname}
 description =
     run tests
-    devdeps: with the latest developer version of key dependencies
     oldestdeps: with the oldest supported version of key dependencies
+    devdeps: with the latest developer version of key dependencies
     online: that require remote data
-setenv =
+pass_env =
+    # A variable to tell tests we are on a CI system
+    CI
+    # Custom compiler locations (such as ccache)
+    CC
+    # Location of locales (needed by sphinx on some systems)
+    LOCALE_ARCHIVE
+    # If the user has set a LC override we should follow it
+    LC_ALL
+set_env =
     MPLBACKEND = agg
+    devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/astropy/simple https://pypi.anaconda.org/scientific-python-nightly-wheels/simple
     COLUMNS = 180
-    PYTEST_COMMAND = pytest -vvv -r fEs --pyargs sunkit_image --cov-report=xml --cov=sunkit_image --cov-config={toxinidir}/.coveragerc {toxinidir}/docs
+    PYTEST_COMMAND = pytest \
```

Comment:
We shouldn't be using this env var any more.

Reply:
Really? we use it here (I can nuke all lines with references to it like 79 and 92) 

theres also a reference in mpl-animators that may need killing too then